src/stores/prompts.js

```diff
@@ -60,20 +60,20 @@ export const usePromptStore = defineStore('prompts', {
       this._isLoading = true
       return await getDoc(doc(db, 'prompts', id))
         .then(async (doc) => {
-          const prompt = { id: doc.id, ...doc.data() }
-          prompt.author = await getDoc(prompt.author).then((doc) => doc.data())
-
-          this._prompts = this.getPrompts
-          const index = this._prompts.findIndex((p) => p.id === prompt.id)
-
-          if (index === -1) this._prompts.push(prompt)
-          else this._prompts[index] = prompt
-
-          return prompt
+          if (doc.data == undefined) {
```

Comment:
The addition of this "if check" should be a good thing in general anywhere "fetchPromptById()" is invoked.